packages/mobile/src/escrow/saga.ts

```diff
@@ -183,6 +190,30 @@ function* getSentPayments() {
   }
 }
 
+export function* fetchReclaimSuggestedFee(action: FetchReclaimTransactionFeeAction) {
+  try {
+    const { paymentID } = action
+    const escrow = yield call(getEscrowContract, web3)
+    const stableToken = yield call(getStableTokenContract, web3)
+    const account = yield call(getConnectedUnlockedAccount)
+
+    Logger.debug(TAG + '@fetchReclaimSuggestedFee', 'Fetching reclaim transaction fee')
+
+    const mockReclaimTx = escrow.methods.revoke(paymentID)
+    const mockTxParams: any = { from: account, gasCurrency: stableToken._address }
+    const gas = new BigNumber(yield mockReclaimTx.estimateGas(mockTxParams))
+    const gasPrice = new BigNumber(yield call(fetchGasPrice))
+    const suggestedFeeInWei = gas.multipliedBy(gasPrice)
+
+    yield put(setReclaimTransactionFee(suggestedFeeInWei.toString()))
+
+    Logger.debug(TAG + '@fetchReclaimSuggestedFee', `New reclaim tx fee is: ${suggestedFeeInWei}`)
+  } catch (e) {
+    Logger.error(TAG + '@fetchReclaimSuggestedFee', 'Error fetching reclaim transaction fee', e)
+    throw e
```

Comment:
what catches this? Our sagas shouldn't propagate errors. Instead you should use `showError` to show a banner here